Dockerfile

```
@@ -42,6 +42,8 @@ RUN : \
 
 FROM node:18-alpine AS runtime
 
+RUN npm install --global pm2
```

Should be version-pinned
```suggestion
RUN npm install --global pm2@5
```